sphinx/make_mode.py

```diff
@@ -194,10 +194,10 @@ def build_latex(self):
 
     def build_latexpdf(self):
         # type: () -> int
-        if self.run_generic_build('latex') > 0:
+        if self.run_generic_build('latexpdf') > 0:
             return 1
-        with cd(self.builddir_join('latex')):
-            return subprocess.call([self.makecmd, 'all-pdf'])
+        print("Build finished; the PDF and LaTeX files are in %s." %
+              self.builddir_join('latex'))
```

Comment:
the py36 test on Travis complains about "missing return statement"

Reply:
The need for explicit returns is one thing I really don't like about `mypy`. I've added this now though 😄